test/unit/ObservationTest.cpp

```diff
@@ -0,0 +1,52 @@
+#include "gtest/gtest.h"
+
+#include "Observations.h"
+
+using namespace EERAModel::Observations;
+
+TEST(TestObservations, TestDurationCalculator)
```

Comment:
I think you should include a comment to state that this is a regression test against the original code: not a unit test.